### ISSUE TITLE
EliteOCRcmd error

### ISSUE THREAD

**Issue by blackshadev** [seeebek/EliteOCR#23]
I got an error after running eliteOCRcmd.exe -i ./test/screenshot_0000.bmp -o test.csv -l eng
The program runs fine with the GUI but I want to include the command line tool in a file watcher and automatically upload the results.

Error:
```
Please make sure the TESSDATA_PREFIX environment variable is set to the parent directory of your "tessdata" directory.
Failed loading language 'big'
Tesseract couldn't load any languages!
AdaptedTemplates != NULL:Error:Assert failed:in file adaptmatch.cpp, line 174
```

**Comment by blackshadev** [comment]
I found the problem. When you call the eliteOCRcmd.exe from a other folder than the eliteOCR folder itself it crashes. Maybe resolve paths based upon the execution path within your commandline tool?

**Comment by seeebek** [comment]
Thanks for the bug report.
Issue solved in version 0.5.3
